changelog.txt

```diff
@@ -1,3 +1,8 @@
+2.6.1-preview
+=============
+- **Setting ForceRefresh = true in AcquireTokenSilent skips access token cache lookup** MSAL now handles ForceRefresh=true correctly and circumvents looking up an access token in the cache, instead using the refresh token to acquire a new access token. [MSAL issue #695](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/695)
```

Comment:
skips => used to skip.
Ensure => Ensured.

It's better to present in bold the end result, rather than the problem (which is already presented in the GitHub  issues) ?